src/components/Table/TableToolbarSearch/_table-toolbar-search.scss

```diff
@@ -0,0 +1,12 @@
+@import '~carbon-components/scss/globals/scss/vars';
```

Comment:
This is not needed, it is imported in the local global vars file that you import on the following line
```suggestion

```